.github/workflows/stale-branches

```diff
@@ -0,0 +1,33 @@
+# .github/workflows/stale-branches.yml
+
+name: Stale Branches
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
+    
+permissions:
+  issues: write
+  contents: write
+
+jobs:
+  stale_branches:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Stale Branches
+      uses: crs-k/stale-branches@v5.0.0
```

Comment:
There's v7 of this action available now - https://github.com/crs-k/stale-branches/releases/tag/v7.0.0

I'd also suggest pinning to the commit id of the version

```suggestion
      uses: crs-k/stale-branches@c6e09a3de1046d68b21eccdca23321d0ec277964 # v7.0.0
```